### PR TITLE
Advertise ipv6 link local address

### PR DESCRIPTION
**Pull request by sumukhatv**
**- What I did**
Modify the neighbor_advertiser script to advertise ipv6 link local address
This PR resolves https://github.com/Azure/sonic-buildimage/issues/6599
**- How I did it**
Add ipv6 link local address to the slice returned by the neighbor advertiser script
**- How to verify it**
Added the verification logic to the neighbor_advertiser test
**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**

### CODE REVIEW

File: scripts/neighbor_advertiser [hunk 1]
```diff
@@ -233,6 +234,21 @@ def get_vlan_addresses(vlan_interface):
...
+        try:
+            out = subprocess.check_output(['ip', '-6', 'addr', 'show', vlan_interface])
+            out = out.decode('UTF-8')
+            for line in out.splitlines():
+                keys = line.split()
+                if keys[0] == 'inet6':
+                    ip = IPNetwork(keys[1])
+                    if str(ip.ip).startswith("fe80"):
+                        ipv6_addr.append(str(ip.ip))
+                        ipv6_prefix.append(ip.prefixlen)
```
Comment (prsunny) [inline scripts/neighbor_advertiser:247]: For link-local, the prefix length should be always set to 128.

File: tests/neighbor_advertiser_test.py [hunk 2]
```diff
@@ -19,6 +20,14 @@ def set_up(self):
         neighbor_advertiser.connect_app_db()
 
     def test_neighbor_advertiser_slice(self, set_up):
+        cmd = "sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0"
+        subprocess.check_output(cmd.split())
+        cmd = "sudo ip link add Vlan1000 type dummy"
+        subprocess.check_output(cmd.split())
+        cmd = "sudo ip -6 address add dev Vlan1000 scope link fe80::1e34:daff:fe1e:2800/64"
+        subprocess.check_output(cmd.split())
+        cmd = "sudo ip link add Vlan2000 type dummy"
```
[the remark this answers is not in the document]
Comment (sumukhatv) [reply tests/neighbor_advertiser_test.py:29]: This is not duplicate. It's for Vlan2000

**Review by prsunny — APPROVED** [review]

**Comment by abdosi** [comment]
@sumukhatv / @prsunny i have cherry-picked in 201911. Test file could not get cherry-picked. Please see if things are fine with python2 and there is no exception.

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-pick to 202012. Please submit another PR.

**Pull request merged**